### PR TITLE
Update math_ops.py with additional usage examples

### PR DESCRIPTION
**Pull request by anigasan**
Added usage examples for tf.math (subtract, scalar_mul, truediv, divide_no_nan, multiply_no_nan, floordiv, reduce_prod, reduce_min, reduce_max, sigmoid, log_sigmoid, unsorted_segmentation, unsorted_segment_sqrt_n, polyval)

### CODE REVIEW

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -3230,6 +3295,11 @@ def log_sigmoid(x, name=None):
 
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
+  
+  For example:
+  >>> x = 32.0
+  >>> tf.math.log_sigmoid(x)
+  <tf.Tensor: shape=(), dtype=float32, numpy=-1.2664166e-14>
```
Comment (alextp) [inline tensorflow/python/ops/math_ops.py]: Can you use a smaller value than 32, so the doctest doesn't need so much precision?

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -3297,9 +3297,8 @@ def log_sigmoid(x, name=None):
   we use `y = -tf.nn.softplus(-x)`.
   
   For example:
-  >>> x = 32.0
-  >>> tf.math.log_sigmoid(x)
-  <tf.Tensor: shape=(), dtype=float32, numpy=-1.2664166e-14>
+  >>> tf.math.log_sigmoid(tf.constant(1.0, tf.float32))
+  <tf.Tensor: shape=(), dtype=float32, numpy=-0.31326166>
```
Comment (alextp) [inline tensorflow/python/ops/math_ops.py]: Use fewer digits of precision (like -0.31...)

File: tensorflow/python/ops/math_ops.py [hunk 3]
```diff
@@ -3298,7 +3298,7 @@ def log_sigmoid(x, name=None):
   
   For example:
   >>> tf.math.log_sigmoid(tf.constant(1.0, tf.float32))
-  <tf.Tensor: shape=(), dtype=float32, numpy=-0.31326166>
+  <tf.Tensor: shape=(), dtype=float32, numpy=-0.31>
```
Comment (alextp) [inline tensorflow/python/ops/math_ops.py]: If you don't add the ... after 0.31 the doctest will fail. Ellipsis are interpreted as wildcard matches.

**Review by alextp — DISMISSED** [review]

File: tensorflow/python/ops/math_ops.py [hunk 4]
```diff
@@ -1163,7 +1163,7 @@ def multiply_no_nan(x, y, name=None):
   For example:
   >>> x = 0.0
   >>> y = 34.0
-  >>> tf.math.multiply_no_nan(0.0, 34.0)
+  >>> tf.math.multiply_no_nan(x, y)
```
Comment (alextp) [inline tensorflow/python/ops/math_ops.py]: I think you should put a NaN as input so people understand that 0 * NaN = 0 for mul_no_nan

File: tensorflow/python/ops/math_ops.py (hunk 3 above)
Comment (anigasan) [reply tensorflow/python/ops/math_ops.py]: Hello, I had made the changes requested. Can you please take a look?

File: tensorflow/python/ops/math_ops.py (hunk 4 above)
Comment (anigasan) [reply tensorflow/python/ops/math_ops.py]: @alextp I added the necessary changes, can you please take a look at the changes that I had made?

**Comment by anigasan** [comment]
@alextp can you take a look at the changes I made?

**Review by alextp — DISMISSED** [review]

File: tensorflow/python/ops/math_ops.py [hunk 5]
```diff
@@ -356,6 +356,27 @@ def _mul(x, y, name=None):
 @tf_export("math.subtract", "subtract")
 @dispatch.add_dispatch_support
 def subtract(x, y, name=None):
+  """Returns 'x' - 'y' element wise
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Please remove all trailing whitespace (tabs and spaces at the end of the line).

You can configure your editor to strip these on save or at least to show them to you so they become visible

File: tensorflow/python/ops/math_ops.py [hunk 6]
```diff
@@ -1446,6 +1493,7 @@ def range(start, limit=None, delta=1, dtype=None, name="range"):  # pylint: disa
 
   For example:
 
+  ```python
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Don't add ` ```python` and ` ``` `, they are not needed

File: tensorflow/python/ops/math_ops.py [hunk 7]
```diff
@@ -1625,16 +1620,13 @@ def reduce_sum_v1(input_tensor,
...
-  x = tf.constant([[1, 1, 1], [1, 1, 1]])
-  tf.reduce_sum(x)  # 6
-  tf.reduce_sum(x, 0)  # [2, 2, 2]
-  tf.reduce_sum(x, 1)  # [3, 3]
-  tf.reduce_sum(x, 1, keepdims=True)  # [[3], [3]]
-  tf.reduce_sum(x, [0, 1])  # 6
-  ```
-
+  >>> x = tf.constant([[1, 1, 1], [1, 1, 1]])
+  >>> tf.reduce_sum(x)  # 6
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: This doesn't follow the doctest format. Should have been

```python
>>> tf.reduce_sum(x)
6
```

and so on

File: tensorflow/python/ops/math_ops.py [hunk 8]
```diff
@@ -1619,12 +1619,9 @@ def reduce_sum_v1(input_tensor,
...
   >>> x = tf.constant([[1, 1, 1], [1, 1, 1]])
-  >>> tf.reduce_sum(x)  # 6
-  >>> tf.reduce_sum(x, 0)  # [2, 2, 2]
-  >>> tf.reduce_sum(x, 1)  # [3, 3]
-  >>> tf.reduce_sum(x, 1, keepdims=True)  # [[3], [3]]
-  >>> tf.reduce_sum(x, [0, 1])  # 6
-  
+  >>> tf.reduce_sum(x)
+  <tf.Tensor: shape=(), dtype=int32, numpy=6>
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: You should keep the other examples too.

File: tensorflow/python/ops/math_ops.py [hunk 9]
```diff
@@ -3359,45 +3386,46 @@ def cumsum(x, axis=0, exclusive=False, reverse=False, name=None):
...
+  >>> tf.cumsum(x, reverse=True) 
   <tf.Tensor: shape=(4,), dtype=int32,
   numpy=array([20, 18, 14,  8], dtype=int32)>
 
   This is more efficient than using separate `tf.reverse` ops.
   The `reverse` and `exclusive` kwargs can also be combined:
-
-  >>> # tf.cumsum([a, b, c], exclusive=True, reverse=True)  # [b + c, c, 0]
+  
+  # tf.cumsum([a, b, c], exclusive=True, reverse=True)  # [b + c, c, 0]
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Why is `>>>` removed?

File: tensorflow/python/ops/math_ops.py [hunk 10]
```diff
@@ -3657,6 +3685,15 @@ def unsorted_segment_mean(data, segment_ids, num_segments, name=None):
 
   If the given segment ID `i` is negative, the value is dropped and will not
   be added to the sum of the segment.
+  
+  For example:
+  >>> x = tf.constant([1, 2, 3, 4])
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Add empty line above.

File: tensorflow/python/ops/math_ops.py [hunk 11]
```diff
@@ -3706,6 +3743,14 @@ def unsorted_segment_sqrt_n(data, segment_ids, num_segments, name=None):
 
   If the given segment ID `i` is negative, the value is dropped and will not
   be added to the sum of the segment.
+  
+  For example:
+  >>> x = tf.constant([1.0, 2.0, 3.0, 4.0])
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Add empty line above.

File: tensorflow/python/ops/math_ops.py [hunk 12]
```diff
@@ -4227,7 +4272,14 @@ def polyval(coeffs, x, name=None):
   evaluated using Horner's method, i.e.
 
      p(x) = coeffs[n-1] + x * (coeffs[n-2] + ... + x * (coeffs[1] +
-            x * coeffs[0]))
+            x * coeffs[0])) 
+  For example:
+  >>> coefficients = [tf.constant(32, dtype=tf.int32),
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Add empty line above.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@anigasan Can you please check mihaimaruseac's comments and keep us posted? Thanks!

**Comment by anigasan** [comment]
Sorry about the lack of response, I am a little pressed with time at the moment. I will get to making the respective changes once I get time

Thanks!

File: tensorflow/python/ops/math_ops.py [hunk 13]
```diff
@@ -3359,45 +3386,46 @@ def cumsum(x, axis=0, exclusive=False, reverse=False, name=None):
   element of the input is identical to the first element of the output:
   For example:
 
-  >>> # tf.cumsum([a, b, c])   # [a, a + b, a + b + c]
+  # tf.cumsum([a, b, c])   # [a, a + b, a + b + c]
```
[the remark this answers is not in the document]
Comment (anigasan) [reply tensorflow/python/ops/math_ops.py]: Thought it was a sidenote since implementation was further into the doctest anyway. Should I keep or no?

File: tensorflow/python/ops/math_ops.py [hunk 14]
```diff
@@ -1725,11 +1728,17 @@ def reduce_euclidean_norm(input_tensor, axis=None, keepdims=False, name=None):
...
+  <tf.Tensor: shape=(), dtype=int32, numpy=4>
+  >>> tf.math.reduce_euclidean_norm(x, 0)
+  <tf.Tensor: shape=(3,), dtype=int32, numpy=array([1, 2, 3], dtype=int32)>
+  >>> tf.reduce_euclidean_norm(x, 1)  
+  <tf.Tensor: shape=(2,), dtype=int32, numpy=array([3, 1], dtype=int32)>
+  >>> tf.math.reduce_euclidean_norm(x, 1, keepdims=True)  
+  <tf.Tensor: shape=(2, 1), dtype=int32, numpy=
+  array([[3],
+        [1]], dtype=int32)>
+  >>> tf.math.reduce_euclidean_norm(x, [0, 1])  # sqrt(17)
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: We can include the output of this in the doctest.

File: tensorflow/python/ops/math_ops.py [hunk 15]
```diff
@@ -2209,7 +2241,11 @@ def reduce_min(input_tensor, axis=None, keepdims=False, name=None):
 
   If `axis` is None, all dimensions are reduced, and a
   tensor with a single element is returned.
-
+  
+  For example:
+  >>> tf.math.reduce_min(tf.constant([32, 36]), axis=None, keepdims=True)
```
[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: The empty line should be between `For example:` and `>>> ...`

File: tensorflow/python/ops/math_ops.py [hunk 16]
```diff
@@ -3386,46 +3437,43 @@ def cumsum(x, axis=0, exclusive=False, reverse=False, name=None):
...
   array([[ 2,  4,  6,  8],
-         [ 3,  7, 11, 15]], dtype=int32)>
-         
+         [ 3,  7, 11, 15]], dtype=int32)>       
   >>> tf.cumsum(y, axis=1)
   <tf.Tensor: shape=(2, 4), dtype=int32, numpy=
   array([[ 2,  6, 12, 20],
          [ 1,  4,  9, 16]], dtype=int32)>
- 
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Trailing whitespace

File: tensorflow/python/ops/math_ops.py (hunk 10 above)
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Empty line added where it shouldn't have. It should be between `For example` and the doctest

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by mihaimaruseac** [comment]
Also, please rebase on master again to solve conflicts

File: tensorflow/python/ops/math_ops.py [hunk 17]
```diff
@@ -356,6 +356,26 @@ def _mul(x, y, name=None):
 @tf_export("math.subtract", "subtract")
 @dispatch.add_dispatch_support
 def subtract(x, y, name=None):
+  """Returns 'x' - 'y' element wise
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Period at the end of the phrase.

File: tensorflow/python/ops/math_ops.py [hunk 18]
```diff
@@ -356,6 +356,26 @@ def _mul(x, y, name=None):
...
+  
+  For example:
+  >>> x = tf.constant([32])
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Empty line between `For example` and the doctest (make sure you do this for _every_ doctest you add in the PR)

File: tensorflow/python/ops/math_ops.py [hunk 19]
```diff
@@ -356,6 +356,26 @@ def _mul(x, y, name=None):
...
+  >>> y = tf.constant([34])
+  >>> tf.math.subtract(x, y)
+  <tf.Tensor: shape=(1,), dtype=float32, numpy=array([-2.], dtype=float32)>
+  
+  Args:
+  'x': A 'Tensor'. Must be one of the following types:
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: These should be indented. See example at `_neg` below. Please make sure to follow the same style throughout the PR. Including for `Returns`.

File: tensorflow/python/ops/math_ops.py [hunk 20]
```diff
@@ -1381,7 +1427,7 @@ def equal(x, y, name=None):
   boolean values.
 
   For example:
-
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Now this looks like a line you changed. Please configure your editor to remove trailing whitespace on save or remove it manually.
LAEO9bhS4sW

File: tensorflow/python/ops/math_ops.py [hunk 21]
```diff
@@ -1484,7 +1529,7 @@ def range(start, limit=None, delta=1, dtype=None, name="range"):  # pylint: disa
   `range(n) = range(0, n)`.
 
   For example:
-
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: LAEO9bhS4sW

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Please make sure docstrings you add are all consistent between themselves and also consistent with other docstrings in the file. Please make sure that you trim trailing whitespace.

I commented only in a few places but you need to go through the entire PR and apply the same changes.

**Comment by gbaned** [comment]
@anigasan Can you please check mihaimaruseac's comments and keep us posted? Thanks!

**Comment by anigasan** [comment]
Apologies! Was busy with various school engagements, but I should be able to work on the changes.

**Comment by gbaned** [comment]
@anigasan can you please resolve conflicts ?

**Comment by anigasan** [comment]
Thanks for the advice @mihaimaruseac, I will make the changes promptly

File: tensorflow/python/ops/math_ops.py [hunk 22]
```diff
@@ -1826,7 +1826,7 @@ def reduce_euclidean_norm(input_tensor, axis=None, keepdims=False, name=None):
         [1]], dtype=int32)>
   >>> tf.math.reduce_euclidean_norm(x, [0, 1])
   <tf.Tensor: shape=(), dtype=int32, numpy=4>
-
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Please don't add trailing whitespace

File: tensorflow/python/ops/math_ops.py [hunk 23]
```diff
@@ -4475,6 +4475,8 @@ def polyval(coeffs, x, name=None):
   >>> tf.math.polyval(coefficients, w)
   <tf.Tensor: shape=(), dtype=int32, numpy=99>
 
+            x * coeffs[0]))
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Is this a typo?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/math_ops.py (hunk 17 above)
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: This is still not solved.

File: tensorflow/python/ops/math_ops.py (hunk 18 above)
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Not solved yet.

File: tensorflow/python/ops/math_ops.py (hunk 19 above)
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Not solved yet

File: tensorflow/python/ops/math_ops.py [hunk 24]
```diff
@@ -437,15 +457,17 @@ def _neg(x, name=None):
...
 @tf_export(v1=["math.scalar_mul", "scalar_mul"])
 def scalar_mul(scalar, x, name=None):
   """Multiplies a scalar times a `Tensor` or `IndexedSlices` object.
 
   Intended for use in gradient code which might deal with `IndexedSlices`
   objects, which are easy to multiply by a scalar but more expensive to
   multiply with arbitrary tensors.
+  
+  For example:
+  >>> tf.math.scalar_mul(32, tf.constant([34,35]))
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Empty line before the doctest starts

File: tensorflow/python/ops/math_ops.py [hunk 25]
```diff
@@ -1136,6 +1158,12 @@ def truediv(x, y, name=None):
   point, the output will have the same type.  If the inputs are integral, the
   inputs are cast to `float32` for `int8` and `int16` and `float64` for `int32`
   and `int64` (matching the behavior of Numpy).
+  
+  For example:
+  >>> num = tf.constant([16, 8])
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Same here

File: tensorflow/python/ops/math_ops.py [hunk 26]
```diff
@@ -1182,6 +1210,12 @@ def div(x, y, name=None):
 @dispatch.add_dispatch_support
 def div_no_nan(x, y, name=None):
   """Computes a safe divide which returns 0 if the y is zero.
+  
+  For example:
+  >>> x = 32.0
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: And here

File: tensorflow/python/ops/math_ops.py [hunk 27]
```diff
@@ -1202,6 +1236,12 @@ def div_no_nan(x, y, name=None):
 @dispatch.add_dispatch_support
 def multiply_no_nan(x, y, name=None):
   """Computes the product of x and y and returns 0 if the y is zero, even if x is NaN or infinite.
+  
+  For example:
+  >>> x = tf.constant(float('nan'))
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: And here

File: tensorflow/python/ops/math_ops.py [hunk 28]
```diff
@@ -1244,6 +1284,12 @@ def floordiv(x, y, name=None):
 
   `x` and `y` must have the same type, and the result will have the same type
   as well.
+  
+  For example:
+  >>> x = 32.0
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: And here

File: tensorflow/python/ops/math_ops.py (hunk 20 above)
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Trailing whitespace not removed

File: tensorflow/python/ops/math_ops.py [hunk 29]
```diff
@@ -1456,7 +1502,6 @@ def not_equal(x, y, name=None):
   of boolean values.
 
   For example:
-
   >>> x = tf.constant([2, 4])
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Why did you remove the empty line? Please add it back

File: tensorflow/python/ops/math_ops.py (hunk 21 above)
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Trailing whitespace is still here

File: tensorflow/python/ops/math_ops.py [hunk 30]
```diff
@@ -1661,16 +1706,21 @@ def reduce_sum_v1(input_tensor,
...
+  <tf.Tensor: shape=(3,), dtype=int32, numpy=array([2, 2, 2], dtype=int32)>
+  >>> tf.reduce_sum(x, 1)
+  <tf.Tensor: shape=(2,), dtype=int32, numpy=array([3, 3], dtype=int32)>
+  >>> tf.reduce_sum(x, 1, keepdims=True)
+  <tf.Tensor: shape=(2, 1), dtype=int32, numpy=
+  array([[3],
+       [3]], dtype=int32)>
+  >>> tf.reduce_sum(x, [0, 1])
+  <tf.Tensor: shape=(), dtype=int32, numpy=6>
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: This example should be after documenting `Args:` and `Returns:`

File: tensorflow/python/ops/math_ops.py [hunk 31]
```diff
@@ -1625,16 +1672,13 @@ def reduce_sum(input_tensor, axis=None, keepdims=False, name=None):
...
-  ```python
-  x = tf.constant([[1, 1, 1], [1, 1, 1]])
-  tf.reduce_sum(x)  # 6
-  tf.reduce_sum(x, 0)  # [2, 2, 2]
-  tf.reduce_sum(x, 1)  # [3, 3]
-  tf.reduce_sum(x, 1, keepdims=True)  # [[3], [3]]
-  tf.reduce_sum(x, [0, 1])  # 6
-  ```
-
+  >>> x = tf.constant([[1, 1, 1], [1, 1, 1]])
```
[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Should also have an empty line before it starts

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/math_ops.py [hunk 32]
```diff
@@ -396,6 +396,27 @@ def _mul(x, y, name=None):
...
+  Args:
+    'x': A 'Tensor'. Must be one of the following types:
+    'bfloat16', 'half', 'float32', 'float64',
+    'uint8', 'int8', 'uint16', 'int16',
+    'int32', 'int64', 'complex64', 'complex128'.
+    'y': A 'Tensor'. Must have the same type as 'x'
+    'name': A name for the operation (optional)
+  
+  Returns:
+  A 'Tensor'. Has same type as 'x'.
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Please indent this line

File: tensorflow/python/ops/math_ops.py [hunk 33]
```diff
@@ -676,11 +703,11 @@ def angle(input, name=None):
 
   For example:
 
-  ```
-  input = tf.constant([-2.25 + 4.75j, 3.25 + 5.75j], dtype=tf.complex64)
-  tf.math.angle(input).numpy()
-  # ==> array([2.0131705, 1.056345 ], dtype=float32)
-  ```
+  >>> input = tf.constant([-2.25 + 4.75j, 3.25 + 5.75j], dtype=tf.complex64)
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: I would recommend using `tf.constant([-1 + j, 2 + 2j], dtype=tf.complex64)` instead as the angles (+/- 45 degrees) are easier to recognize. Plus, then the line won't be extremely long.

File: tensorflow/python/ops/math_ops.py [hunk 34]
```diff
@@ -2171,7 +2203,7 @@ def reduce_min_v1(input_tensor,
 
   If `axis` is None, all dimensions are reduced, and a
   tensor with a single element is returned.
-
+
```
[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Trailing whitespace not removed.

Screenshot from 2020-02-18 09-52-11

File: tensorflow/python/ops/math_ops.py [hunk 35]
```diff
@@ -2290,6 +2326,10 @@ def reduce_max(input_tensor, axis=None, keepdims=False, name=None):
 
   If `axis` is None, all dimensions are reduced, and a
   tensor with a single element is returned.
+  
+  For example:
+  >>> tf.math.reduce_max(tf.constant([32.0, 36.0]), axis=None, keepdims=True)
```
[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply tensorflow/python/ops/math_ops.py]: Empty line not added.

File: tensorflow/python/ops/math_ops.py [hunk 36]
```diff
@@ -2839,7 +2950,7 @@ def matmul(a,
          [139, 154]], dtype=int32)>
 
   A batch matrix multiplication with batch shape [2]:
-
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Trailing whitespace. Please configure your editor/IDE to trim whitespaces on save.

File: tensorflow/python/ops/math_ops.py [hunk 37]
```diff
@@ -3338,6 +3452,9 @@ def sigmoid(x, name=None):
   If a negative number is large, its sigmoid will approach to 0 since the
   formula will be `y = 1 / (1 + <large_num>)`
 
+  Specifically, `y = 1 / (1 + exp(-x))`.
+  
+  For example:
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/math_ops.py]: Missing empty line

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@anigasan Can you please check mihaimaruseac's comments and keep us posted? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by anigasan** [comment]
Yes, this is still being worked on, haven't had time to work on it due to numerous school exams. Will get to it soon

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by anigasan** [comment]
Yes this is still being worked on, will let you know if other changes need to be made

**Comment by gbaned** [comment]
@anigasan  Any update on this PR, please. Thanks!

**Comment by anigasan** [comment]
I will see whether I have time to work on this, I apologize for the delay. I have been very busy with schoolwork and exams. Will let you know and make changes accordingly

**Comment by mihaimaruseac** [comment]
We'll be closing the PR for now to let others also work on it. When you get a chance to work on it please reopen
